src/tools/annotation/EllipticalRoiTool.js

```diff
@@ -238,7 +238,17 @@ export default class EllipticalRoiTool extends BaseAnnotationTool {
         const ellipseOptions = { color };
 
         if (renderDashed || data.hasDashedBorder) {
-          ellipseOptions.lineDash = lineDash;
+          const dashSpacing = data.lineDash;
```

Comment:
The `lineDash` property is too vague - especially when adding properties to a library that's used, you want the code to be self-documenting. Can you rename `lineDash` to something like `spaceBetweenDashes`?